### PR TITLE
Include instructions for snaps

### PR DESCRIPTION
**Pull request by imjustmatthew**
Include instructions for installing snapd and qubes-snapd-helper from issue #2766 (https://github.com/QubesOS/qubes-issues/issues/2766) based on the instructions from @ioleo that worked for me.

### CODE REVIEW

**Comment by imjustmatthew** [comment]
I'm not really sure why this is showing as build failed, it looks like there's some kind of content check?

File: user/common-tasks/software-update-domu.md [hunk 1]
```diff
@@ -198,6 +198,70 @@ Example policy file in R4.0 (with Whonix installed, but not set as default Updat
...
+Last metadata expiration check: 0:57:08 ago on Thu Nov 14 09:26:47 2019.
+Notifying dom0 about installed applications
+
+Installed:
+  snapd-2.42.1-1.fc30.x86_64                                              qubes-snapd-helper-1.0.1-1.fc30.noarch                           
+[...]                          
+Complete!
+```
+You may see a message "Failed to resolve booleanif statement at /var/lib/selinux/targeted/tmp/modules/200/snappy/cil:1174
+/usr/sbin/semodule:  Failed!". This is expected and you cna safely continue.
```
Comment (hexagonrecursion) [inline user/common-tasks/software-update-domu.md]: > This is expected and you cna safely continue.

s/cna/can/

**Comment by andrewdavidwong** [comment]
> I'm not really sure why this is showing as build failed, it looks like there's some kind of content check?

Yes, it checks for unknown spellings. You can see this if you click through to the full log and scroll down toward the bottom. In particular, this section:

```
The following new unknown spellings were introduced:
    +Applicaitons
    +availble
    +booleanif
    +fc30
    +persistant
    +r4
    +snapd
```

At least these appear to be actual typos:

```
    +Applicaitons
    +availble
    +persistant
```

**Comment by imjustmatthew** [comment]
Thanks. @andrewdavidwong the second commit in this branch fixes those issues, clarifies 2 sentences, and fixes the issue pointed out by @hexagonrecursion The current build fails with these new spellings, which I believe are all correct:

```
The following new unknown spellings were introduced:
    +booleanif
    +fc30
    +r4
    +snapd
```

File: user/common-tasks/software-update-domu.md [hunk 2]
```diff
@@ -198,6 +198,70 @@ Example policy file in R4.0 (with Whonix installed, but not set as default Updat
...
+[user@fedora-30-snap-demo ~]$ sudo shutdown -h now
+```
+
+2. Now open the **AppVM** in which you would like to install the Snap application and run a terminal:
+```shell_session
+[user@snap-demo-AppVM ~]$ snap install <package>
+```
+when the install is complete shutdown the AppVM:
+```shell_session
+[user@snap-demo-AppVM ~]$ sudo shutdown -h now
```
Comment (marmarek) [inline user/common-tasks/software-update-domu.md]: Shutting down AppVM isn't necessary step. Especially, the next step (refreshing application list) will start it again.

**Comment by imjustmatthew** [comment]
Requested changes have been implemented in the latest commit to this branch.

**Pull request merged**